x-pack/plugins/spaces/public/management/management_service.tsx

```diff
@@ -25,12 +25,11 @@ export class ManagementService {
   private registeredSpacesManagementApp?: ManagementApp;
 
   public setup({ getStartServices, management, spacesManager, securityLicense }: SetupDeps) {
-    const kibanaSection = management.sections.getSection('kibana');
-    if (kibanaSection) {
-      this.registeredSpacesManagementApp = kibanaSection.registerApp(
+    this.registeredSpacesManagementApp = management.sections
```

Comment:
If we're assuming that the `kibana` section will always exist, can we update the type definitions to reflect that, rather than forcing TypeScript into trusting us here?